### DISCUSSION TITLE
使用xmake安装openblas会失败

### DISCUSSION THREAD

**Discussion by yeruilin** [xmake-io/xmake#3983] [General]
具体来说，xrepo install openblas会显示error: execv(xmake require -j 14 --extra={system=false} openblas) failed(-1)的错误，不知道有其他人遇到过这个问题吗？

**Comment by yeruilin** [comment]
还有另一个报错信息，cannot move bin\libopenblas.dll to D:\packages\xmake_repo\o\openblas\0.3.21\e582adfc053645e4aaaf3338753d86db\bin\libopenblas.dll Unknown

**Reply by waruqi** [reply]
再试试

xrepo update-repo
xmake f -c
xmake

**Reply by yeruilin** [reply]
嗯嗯，解决了，多谢多谢！
